app/components/submission/header.js

```diff
@@ -400,7 +411,7 @@ export default class SubmissionHeaderComponent extends Component {
       const submissions = await internalReviewOfSubcase.hasMany('submissions').reload();
       addObject(submissions, this.args.submission);
       internalReviewOfSubcase.submissions = submissions;
-      return await this.args.submission.save();
+      return await internalReviewOfSubcase.save();
```

Comment:
this was wrong, BIS submission internal review were not saved because the wrong model was saved